### PR TITLE
feat(StructuralModel): first implementation of Structural Model with faults and horizons

### PR DESCRIPTION
**Pull request by panquez**

### CODE REVIEW

File: src/geode/geosciences/core/structural_model.cpp [hunk 1]
```diff
@@ -21,15 +21,10 @@
  *
  */
 
-#include <mylib/hello_world.h>
+#include <geode/geosciences/core/structural_model.h>
```
Comment (BotellaA) [inline src/geode/geosciences/core/structural_model.cpp]: Is this file useful?

Comment (panquez) [reply src/geode/geosciences/core/structural_model.cpp]: Removed

File: azure-pipelines.yml [hunk 2]
```diff
@@ -129,7 +129,19 @@ jobs:
...
+- template: documentation.yml@templates
+  parameters:
+    dependencies:	
+    - Semantic_release
+    repo: "OpenGeode-Geosciences"
+    preCMake:
+    - template: get-release.yml
+      parameters:
+        repo: OpenGeode
+        pattern: "*Linux*"
```
[the remark this answers is not in the document]
Comment (panquez) [reply azure-pipelines.yml]: Done for all repo

File: README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+<h1 align="center">OpenGeode-Geosciences<sup><i>by Geode-solutions</i></sup></h1>
+<h3 align="center">OpenGeode module for Geosciences</h3>
 
 <p align="center">
-  <img src="https://dev.azure.com/GeodeSolutions/Geode/_apis/build/status/Geode-solutions.OpenGeode-ModuleTemplate?branchName=master" alt="Build Status">
-  <img src="https://img.shields.io/azure-devops/tests/GeodeSolutions/Geode/10/master.svg?compact_message" alt="Test Status">
-  <img src="https://img.shields.io/azure-devops/coverage/GeodeSolutions/Geode/10/master.svg" alt="Coverage Status">
-  <img src="https://img.shields.io/github/release/Geode-solutions/OpenGeode-ModuleTemplate.svg" alt="Version">
+  <img src="https://dev.azure.com/GeodeSolutions/Geode/_apis/build/status/Geode-solutions.OpenGeode-Geosciences?branchName=master" alt="Build Status">
+  <img src="https://img.shields.io/azure-devops/tests/GeodeSolutions/Geode/13/master.svg?compact_message" alt="Test Status">
```
[the remark this answers is not in the document]
Comment (panquez) [reply README.md]: Yes it is OK : Azure DevOps Pipeline 13 is still working

**Pull request merged**